src/MergeContext.js

```diff
@@ -1016,10 +1018,13 @@ class PullRequest {
             statusChecks.addOptionalStatus(new StatusCheck(st));
         }
```

Comment:
_getPrStatuses() and _getStagingStatuses() look almost the same now, except for this dealing with 'copied' statuses (although the diff does not show it clearly).